src/loaders/JsonImageInfoLoader.ts

```diff
@@ -308,10 +309,16 @@ class JsonImageInfoLoader extends ThreadableVolumeLoader {
       }
 
       // extract the data
+      const rawRange: [number, number][] = [];
```

Comment:
I feel like `data` (or `channel`) is a better prefix than `raw` here... raw implies to me that some postprocessing will be done to make it less raw later.